python/ray/log.py

```diff
@@ -0,0 +1,195 @@
+import functools
```

Comment:
Don't create log.py and can you just merge it into ray_logging.py? It is confusing we have 2 files like this